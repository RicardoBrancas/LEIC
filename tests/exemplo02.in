A LIS 20
A LAX 10
A OPO 30
F LAX LIS
F LIS OPO
G LAX OPO
G OPO NCE
G NCE LIS
G LIS LAX
F LIS LAX
F OPO LAX
F OPO LAX
I LAX 20
I LIS -15
N LIS LAX
V
G LIS NCE
P
Q
X