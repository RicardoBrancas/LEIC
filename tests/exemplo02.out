*Impossivel adicionar voo OPO NCE
*Impossivel adicionar voo NCE LIS
*Capacidade de LIS inalterada
Voos entre cidades LIS:LAX:3:2
Voo mais popular LIS:LAX:3
*Impossivel adicionar voo LIS NCE
Aeroporto com mais rotas LAX:5:5
Aeroporto com mais ligacoes LIS:2
12:3
